pkg/dispatchserver/main.go

```diff
@@ -23,7 +23,13 @@ var dispatchConfigPath = ""
 
 // NewCLI creates cobra object for top-level Dispatch server
 func NewCLI(out io.Writer) *cobra.Command {
-	log.SetOutput(out)
+	logFile, err := os.OpenFile("./dispatch.log", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```

Comment:
The path to this file should be configurable and it should be possible to disable logging to this file, which personally I think should be the default.

Reply:
Do you think we can add a flag, for example `--logfile filepath`

Reply:
I think that is reasonable.